azure/tlevels-environment.json

```diff
@@ -526,6 +535,49 @@
                 }
             }
         },
+        {
+            "condition": "[equals(parameters('enableReplica'), true())]",
```

Comment:
![image](https://github.com/SkillsFundingAgency/tl-results-and-certification/assets/81469431/d125a102-35ba-4971-a7ba-817ca64e0cf5)
https://learn.microsoft.com/en-us/azure/azure-resource-manager/templates/template-functions-comparison#equals
So it looks like you still need `,true()` because the equals function  needs something to compare enableReplica to.